esrally/track/loader.py

```diff
@@ -1487,17 +1487,17 @@ def parse_parallel(self, ops_spec, ops, challenge_name):
                     )
 
         if completed_by:
-            completion_task = None
+            completion_task = False
```

Comment:
Nit: Based on the variable name it's not clear that this is a flag. How about `has_completion_task`?